src/Game/Game.cpp

```diff
@@ -448,5 +446,19 @@ Audio::Mixer* Game::mixer()
     return _mixer.get();
 }
 
+Event::Dispatcher& Game::eventDispatcher()
+{
+    return _eventDispatcher;
+}
+
+}
+
+namespace Event
+{
+// static
+Dispatcher* Dispatcher::getInstance()
+{
+    return &Game::getInstance()->eventDispatcher();
+}
```

Comment:
There are two issues as I see.
First is an access to EventDispatcher instance from Emitter.
As I think it would be better to specify a dispatcher for every Emitter. But it can be a complex task.

Second, is a control over EventDispatcher for Game instance.
If it will be a standalone singleton then it can be hard to manage it's state befor or after Game lifetime.
My solution isn't perfect but it's a compromise.
And I agree with your opinion about singletons.